core-tests/core-tests.cabal

```diff
@@ -50,3 +50,9 @@ executable failing-pattern-test
   build-depends:       base <= 5, tasty, tasty-hunit, random >= 1.2, mtl
   default-extensions:  ScopedTypeVariables
   ghc-options:         -Wall -fno-warn-type-defaults
+
+executable multiple-pattern-test
+  import:              commons
+  main-is:             multiple-pattern-test.hs
+  build-depends:       base <= 5, tasty, tasty-hunit
```

Comment:
PVP upper bounds need to be strict, so, `base < 5`.
Why? Because the current constraint allows `base-5`, but not e.g. `base-5.0.0.0`, which does not make any sense.  The intention is clearly to only allow `base-4.*`.
This seems to be a systematic error in this file, so blame goes to someone else.

Reply:
Changed.

Reply:
Thanks!